openpower/package/sb-signing-utils/sb-signing-utils.mk

```diff
@@ -0,0 +1,47 @@
+################################################################################
+#
+#  sb-signing-utils
+#
+################################################################################
+
+SB_SIGNING_UTILS_SITE = ssh://git@github.com/hellerda/sb-signing-utils.git
```

Comment:
Before we merge this into op-build, we should get yoru sb-signing-utils changes into the open-power repository for them and point the build script there, using the github magic htat buildroot provides